tests/performance/log-results.js

```diff
@@ -11,8 +11,12 @@ const { median } = require( './utils' );
 
 // The list of test suites to log.
 const testSuites = [
+	'admin',
```

Comment:
Do you know if we need to add these to the [codevitals dashboard](https://www.codevitals.run/project/wordpress)?

Reply:
Depends on whether we want to add them there or not (I'd say yes, otherwise what's the point). We would need to ask Riad to update the dashboard.

Reply:
Right, I was thinking the same. Let's make sure to reach out after these get added.